app/helpers/blacklight/blacklight_maps_helper_behavior.rb

```diff
@@ -27,7 +27,8 @@ def link_to_bbox_search bbox_coordinates
     coords_for_search = bbox_coordinates.map { |v| v.to_s }
     link_to(t('blacklight.maps.interactions.bbox_search'),
             catalog_index_path(spatial_search_type: "bbox",
-                               coordinates: "[#{coords_for_search[1]},#{coords_for_search[0]} TO #{coords_for_search[3]},#{coords_for_search[2]}]"))
+                               coordinates: "[#{coords_for_search[1]},#{coords_for_search[0]} TO #{coords_for_search[3]},#{coords_for_search[2]}]",
+                               view: default_document_index_view_type))
```

Comment:
Could this also be tested in `blacklight_maps_helper_spec.rb`? So that if anything changes down the line we know to expect this.